src/redux/modules/ayahs.js

```diff
@@ -33,8 +40,10 @@ export default function reducer(state = initialState, action = {}) {
         loading: true
       };
     case LOAD_SUCCESS:
+      let current = state.current === null ? action.result.result[0] : state.current;
```

Comment:
jazak allahu khayran i'll fix it insha allah

On Wed, Apr 6, 2016 at 7:13 PM Mohamed El Mahallawy <
notifications@github.com> wrote:

> In src/redux/modules/ayahs.js
> <https://github.com/quran/quran.com-frontend/pull/252#discussion_r58744688>
> :
>
> > @@ -33,8 +40,10 @@ export default function reducer(state = initialState, action = {}) {
> >          loading: true
> >        };
> >      case LOAD_SUCCESS:
> > +      let current = state.current === null ? action.result.result[0] : state.current;
>
> this is weird. it should be let current = state.current ? state.current :
> action.result.result[0]
>
> —
> You are receiving this because you authored the thread.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/quran/quran.com-frontend/pull/252/files/17e999ee178d78dcf971d61df4ef4be5836d5bc9#r58744688>
>